src/icons/PrismOrb.tsx

```diff
@@ -27,147 +27,76 @@ export default function () {
       fill="none"
-      className={svgClasses}
       xmlns="http://www.w3.org/2000/svg"
+      className={svgClasses}
     >
       <path
         d="M2 223.856H130H194H258L130 175.675L2 223.856Z"
         className={prismStroke}
       />
-      <g filter="url(#filter0_d_81_856)">
-        <mask
-          id="mask0_81_856"
-          style={{ maskType: 'alpha' }}
-          maskUnits="userSpaceOnUse"
-          x="83"
-          y="104"
-          width="94"
-          height="95"
-        >
-          <circle
-            cx="129.661"
-            cy="151.66"
-            r="46.6597"
-            transform="rotate(-180 129.661 151.66)"
-            fill="#C4C4C4"
-          />
-        </mask>
-        <g mask="url(#mask0_81_856)">
-          <circle
-            cx="129.63"
-            cy="151.629"
-            r="46.729"
-            transform="rotate(-180 129.63 151.629)"
-            fill="url(#paint0_radial_81_856)"
-          />
-          <circle
-            cx="129.63"
-            cy="151.629"
-            r="46.729"
-            transform="rotate(-180 129.63 151.629)"
-            fill="url(#paint1_radial_81_856)"
-          />
-          <circle
-            cx="129.63"
-            cy="151.629"
-            r="46.729"
-            transform="rotate(-180 129.63 151.629)"
-            fill="url(#paint2_radial_81_856)"
-          />
-          <mask
-            id="mask1_81_856"
-            style={{ maskType: 'alpha' }}
-            maskUnits="userSpaceOnUse"
-            x="82"
-            y="104"
-            width="95"
-            height="95"
-          >
-            <circle
-              cx="129.638"
-              cy="151.628"
-              r="46.729"
-              transform="rotate(-180 129.638 151.628)"
-              fill="white"
-            />
-          </mask>
-          <g mask="url(#mask1_81_856)">
-            <circle
-              cx="129.638"
-              cy="151.628"
-              r="46.729"
-              transform="rotate(-180 129.638 151.628)"
-              fill="url(#paint3_radial_81_856)"
-            />
-            <rect
-              x="152.344"
-              y="109.477"
-              width="111.687"
-              height="102.222"
-              transform="rotate(123.478 152.344 109.477)"
-              fill="url(#paint4_radial_81_856)"
-            />
-          </g>
-          <g style={{ mixBlendMode: 'soft-light' }}>
-            <rect
-              x="129.625"
-              y="211.191"
-              width="53.6397"
-              height="52.6524"
-              transform="rotate(-180 129.625 211.191)"
-              fill="url(#paint5_radial_81_856)"
-            />
-          </g>
-          <g style={{ mixBlendMode: 'soft-light' }}>
-            <path
-              d="M197.094 160.184L122.393 160.184L122.393 86.7999L197.094 86.7999L197.094 160.184Z"
-              fill="url(#paint6_radial_81_856)"
-            />
-          </g>
-        </g>
+      <circle
```

Comment:
I exported the `svg` without `dropShadow` and used our own `dropShadow` (which is above in `svgClasses`). Result:
![image](https://user-images.githubusercontent.com/49658988/174146282-9f1f9e1b-e9b1-498d-af7a-122db52bf58a.png)
was: 
![image](https://user-images.githubusercontent.com/49658988/174146339-aeef54f5-a865-4483-9208-e6bbd15539e3.png)

Reply:
I fixed this black outline